Patch by Lubomir Rintel <lkundrak@v3.sk> for arj >= 3.10.22, which disables
the custom printf to avoid conflicting strnlen definition with the glibc
headers. By using custom printf (as in the past), we're completely loosing
all the _FORTIFY_SOURCE printf protections.

--- arj-3.10.22/fardata.c		2004-04-17 13:39:42.000000000 +0200
+++ arj-3.10.22/fardata.c.printf	2009-04-18 16:23:52.000000000 +0200
@@ -13,7 +13,6 @@
 /* ASR fix 02/05/2003: need that regardless of COLOR_OUTPUT to support -jp
    correctly */
 #if SFX_LEVEL>=ARJ
- #define CUSTOM_PRINTF
  #define CHUNK_SIZE               512    /* Size of the output block */
  #define CHUNK_THRESHOLD (CHUNK_SIZE-256) /* Safety bound */
 #endif
